File: arbol_binario/src/arbol_binario/Nodo.java
package arbol_binario;

public class Nodo
{
	String Dato;
	int Indice;
	Nodo izq = null;
	Nodo der = null;

	public Nodo(int indice) 
	{
		this.Indice=indice;
		this.Dato=null;
		this.izq=null;
		this.der=null;
	}
		
	
}

File: 1-8 Puzzle-main/Puzle1-8/Nodo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Puzle1_8
{
    class Nodo
    {
        int num;
        Nodo izq, der;
        public Nodo()
        {
        }
        public Nodo(int num)
        {
            this.num = num;
            this.izq = null;
            this.der = null;
        }
        public int Num
        {
            get
            {
                return num;
            }
            set
            {
                num = value;
            }
        }
        public Nodo hIzq
        {
            get
            {
                return izq;
            }
            set
            {
                izq = value;
            }
        }
        public Nodo hDer
        {
            get
            {
                return der;
            }
            set
            {
                der = value;
            }
        }
    }
}
